### PR TITLE
Add Movo Coin

### PR DESCRIPTION
**Pull request by movosmartchain**

### CODE REVIEW

**Comment by nnnnnnsssss** [comment]
Reechek

**Comment by movosmartchain** [comment]
> Add Movo Coin

All ok

**Comment by movosmartchain** [comment]
> Reechek

Can you tell me how to upload logo using Ipfs? I have tried it many times
https://ipfs.io/ipfs/QmYG3EEyXGW5v3YmkbuSJnmATa45Fjym2vvLg6oAthYLNR

**Comment by movosmartchain** [comment]
Help me check it @ligi i
thank you!!

File: _data/chains/eip155-2049.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "chainId": 2049,
+  "networkId": 2049,
+  "slip44": 2050,
+  "icon": "movo",
+  "explorers": [
+    {
+      "name": "movoscan",
+      "url": "https://movoscan.com",
+      "icon": "movoscan",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-2049.json:26]: does not seem to be EIP3091 compatible
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-2049.json (hunk 1 above)
Comment (movosmartchain) [reply _data/chains/eip155-2049.json:26]: Thank you!!

Comment (movosmartchain) [reply _data/chains/eip155-2049.json]: I have updated it again

Comment (ligi) [reply _data/chains/eip155-2049.json]: but u did it wrong - not nonce - but "none" - would have been easier if you just committed my suggestion

Comment (movosmartchain) [reply _data/chains/eip155-2049.json]: Sorry, i updated

**Review by ligi — APPROVED** [review]

**Pull request merged**